src/dom_components/view/ComponentTextView.js

```diff
@@ -82,7 +82,7 @@ module.exports = ComponentView.extend({
         }
 
         // Avoid re-render on reset with silent option
-        model.set('content', '');
+        model.set('content', '').trigger('change:content', model);
```

Comment:
@artf What do you think about explicitly triggering the `change:content` event instead of setting it to a random value? I think this a bit more clear what's going on, but I'd be happy to change it like your suggestion if you prefer.

Reply:
Seems definitely less hacky 😬